iroha/Cargo.toml

```diff
@@ -32,6 +32,7 @@ rand = "0.7.3"
 ursa = "0.3.2"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+systemstat = "0.1.5"
```

Comment:
Do we have an approval to use this dependency?

Reply:
Waiting for Makoto's review

Reply:
It seems that this won't work on macOS?
https://github.com/myfreeweb/systemstat/issues/73

Is it possible to use a more mature library? https://github.com/heim-rs/heim/blob/master/COMPARISON.md

Reply:
Strange, I was looking for cross platform solutions but missed that issue - will make a research

Reply:
@takemiyamakoto @eadventurous @vmarkushin are you okay with https://github.com/heim-rs/heim ?

Reply:
Sounds good to me.

Reply:
@humb1t I'm ok with it. I also would like to mention, that this crate provides [features](https://github.com/heim-rs/heim/blob/master/heim/Cargo.toml#L41) that can be disabled in case we don’t need the full system info